src/network-config.js

```diff
@@ -66,6 +66,25 @@ export const networkConfigs = {
       live: true,
     },
   },
+  [chains.getChainInformation(10).type]: {
+    isActive: true,
+    addresses: {
+      ensRegistry:
+        localEnsRegistryAddress || '0x6f2CA655f58d5fb94A08460aC19A552EB19909FD',
+      dai: '0xdc31ee1784292379fbb2964b3b9c4124d8f89c60',
```

Comment:
you used the wrong DAI contract
```suggestion
      dai: '0xda10009cbd5d07dd0cecc66161fc93d7c9000da1',
```

Reply:
Good catch!